### ISSUE TITLE
Black Taskbar

### ISSUE THREAD

**Issue by TrueBlackout** [TranslucentTB/TranslucentTB#535]
I install translucentTB, and it works fine until i restart my computer, once it's restarted the taskbar is black, if i close translucentTB while it's black it becomes transparent, re open it and it becomes black again. I have been unable to find a solution anywhere else.

**Comment by sylveon** [comment]
Are you on insider builds? #531

**Comment by TrueBlackout** [comment]
i do not think so, i have just downloaded the most recent version from the microsoft store and i am on the most recent version of windows 10

**Comment by AgrockP** [comment]
> Are you on insider builds? #531

I am, and no options work

**Comment by DannyCtinescu** [comment]
The same behaviot in insider 22579 _ Taskbar black, no matters which option is checked (blur, transparent, etc)

**Comment by AliHussain332** [comment]
> The same behaviot in insider 22579 _ Taskbar black, no matters which option is checked (blur, transparent, etc)

me too nothing works only black taskbar.

**Comment by AliHussain332** [comment]
> > The same behaviot in insider 22579 _ Taskbar black, no matters which option is checked (blur, transparent, etc)
> 
> me too nothing works only black taskbar.

I checked taskbarx too and it has the same problem.

**Comment by Moonbladelmao** [comment]
yep, same issue for me

**Comment by sneazy-ibo** [comment]
> > Are you on insider builds? #531
> 
> I am, and no options work

Same here TwT

**Comment by masaoka-shiki** [comment]
I had this issue, I just went into task manager, end task and restarted the app and it works fine for a while, idk what casues it

**Comment by TrueBlackout** [comment]
as far as i can tell it seems to only occur for me when my laptop isn't plugged in

**Comment by Quzoide** [comment]
Make sure your transparency effects are turned on in windows. Most likely your power plan has them disabled when your laptop isnt plugged in.

**Comment by TrueBlackout** [comment]
they are turned on in windows and looking through battery saver setting si don't see anything about it disabling transparency

**Comment by lucakato** [comment]
I believe it happens when you turn on battery saver mode. @TrueBlackout can you right click on TranslucentTB via tray icon, then enable Battery Saver. Then set it to Clear instead of Opaque. This seems to make it clear rather than black for me.

**Comment by TrueBlackout** [comment]
that did it! thank you

**Comment by tpmeader** [comment]
So this app used to work wonderfully on all of my computers. I have six laptops, all running windows 11 pro/enterprise and as of the last couple dev builds that have been published, TB has stopped working all together, excepting that it is running in the system tray. But even with it running, I get no context menu via right or left clicking. I try to open the app from the Microsoft store and the Start menu and all that comes up is TB is already running in your system tray. No amount of resets on the app or reinstalling has changed this. Any ideas for a fix would be welcome.

Thank you very much.

**Comment by sylveon** [comment]
Your issue is #537

**Comment by albydaddy22** [comment]
I had the same problem. I fixed it when I turned off the "battery saver" option in the ttb settings. I recommend to enable it and select your favourite setting (clear, acrylic, normal etc.). All of that happened beacause i had the windows battery saver when my laptop's not plugged in.
